2-mandatory/1-weather-report.js

```diff
@@ -13,6 +13,11 @@
 
 function getTemperatureReport(cities) {
     // TODO
+    let array = []
+    for ( let city of cities){
+        array.push(`The temperature in ${city} is ${temperatureService(city)} degrees`);
+    }
+    return array;
```

Comment:
really nice solution & good use of template literals. Could you think of a more explanatory variable name to use here rather than `array`?